aten/src/ATen/Context.cpp

```diff
@@ -66,7 +66,7 @@ bool Context::deterministic() const {
 
 void Context::setDeterministic(bool b) {
```

Comment:
"setDeterministic" -> "setDeterministicAlgorithms" to be consistent with the new naming?

Reply:
Also instead of `_deterministic` maybe `_deterministic_algorithms` or `_deterministic_algos`?